src/remembrall/classes/Record.py

```diff
@@ -35,6 +37,29 @@ def add_birthday(self, birthday):
         if not self.birthday:
             self.birthday = Birthday(birthday)
 
+    def add_address(self, address):
+        if address:
```

Comment:
тут тобі не треба перевіряти на параметр, це буде зловлено в операційних функціях. Тут мало бути так, селв забув:)
```suggestion
        if self.address:
```